### PR TITLE
adding function to read in cmp file and convert to pandas df

### PR DESCRIPTION
**Pull request by daphnecor**

### CODE REVIEW

File: pyaldata/utils.py [hunk 1]
```diff
@@ -543,3 +543,47 @@ def extract_interval_from_signal(trial_data, signal, epoch_fun):
...
+        .cmp file path + name 
+        
+    Returns
+    -------
+    df_array: dataframe of shape (num electrodes, 5)
+        [col (int), row (int), channel number (str), within_channel_num (int), global electrode number (int)]
+    """
+    # Open file, remove comments and remove other formatting we don't need
+    f = open(file_path)
+    temp = [line for line in f.readlines() if not line.startswith('//')]
```
Comment (bagibence) [inline pyaldata/utils.py:582]:
```suggestion
    with open(file_path) as f:
        temp = [line for line in f if not line.startswith('//')] 
```
This closes the file after you're done reading from it.

File: pyaldata/utils.py [hunk 2]
```diff
@@ -543,3 +543,47 @@ def extract_interval_from_signal(trial_data, signal, epoch_fun):
...
+        
+    Returns
+    -------
+    df_array: dataframe of shape (num electrodes, 5)
+        [col (int), row (int), channel number (str), within_channel_num (int), global electrode number (int)]
+    """
+    # Open file, remove comments and remove other formatting we don't need
+    f = open(file_path)
+    temp = [line for line in f.readlines() if not line.startswith('//')] 
+    clean_lsts = [remove_cmp_formatting(l) for i, l in enumerate(temp) if not i == 0]
```
Comment (bagibence) [inline pyaldata/utils.py:583]:
```suggestion
    clean_lsts = [remove_cmp_formatting(l) for l in temp[1:]]
```
You just want to skip the first line, right?

**Review by bagibence — COMMENTED** [review]
Thank you!
I don't have any .cmp file and I'm not sure what they look like, so I just left two suggestions to make the code a bit simpler. Can you check if they work for you?

File: pyaldata/utils.py (hunk 2 above)
Comment (daphnecor) [reply pyaldata/utils.py:583]: correct

**Comment by daphnecor** [comment]
Thanks for the suggestions, the code works as it should!

**Review by bagibence — APPROVED** [review]

**Pull request merged**